static/src/javascripts-legacy/projects/common/modules/experiments/acquisition-test-selector.js

```diff
@@ -1,24 +1,34 @@
 define([
     'common/modules/experiments/segment-util',
+    'common/modules/commercial/acquisitions-view-log',
     'common/modules/experiments/tests/contributions-epic-brexit',
     'common/modules/experiments/tests/contributions-epic-always-ask-strategy',
+    'common/modules/experiments/tests/contributions-epic-ask-four-stagger',
+    'common/modules/experiments/tests/contributions-epic-ask-four-earning'
 ], function (
     segmentUtil,
+    viewLog,
     brexit,
-    alwaysAsk
+    alwaysAsk,
+    askFourStagger,
+    askFourEarning
 ) {
     /**
      * acquisition tests in priority order (highest to lowest)
      */
-    var tests = [alwaysAsk, brexit];
+    var tests = [alwaysAsk, askFourEarning, brexit, askFourStagger];
 
     return {
         getTest: function() {
             var eligibleTests = tests.filter(function (test) {
                 var t = new test();
                 var forced = window.location.hash.indexOf('ab-' + t.id) > -1;
+                var variant = segmentUtil.variantFor(t);
+                var acceptableViewCount = variant? viewLog.viewsInPreviousDays(variant.maxViews.days, t) < variant.maxViews.count : false;
+                var acceptableTimeBetweenViews = variant? viewLog.viewsInPreviousDays(variant.maxViews.minDaysBetweenViews, t) === 0 : false;
 
-                return forced || (t.canRun() && segmentUtil.isInTest(t));
+
+                return forced || (t.canRun() && segmentUtil.isInTest(t) && acceptableViewCount && acceptableTimeBetweenViews);
```

Comment:
maybe look into using isInTest in a different way because we'll end up calling `variantForId` more than once here (from `isInTest` and `segmentUtil.variantFor(t)` above)